ide/coqide/coqOps.ml

```diff
@@ -759,7 +760,7 @@ object(self)
            | Some loc ->
              let (iter, _, phrase) = self#get_sentence s in
              let (start, _) = Loc.unloc loc in
-             iter#forward_chars (b2c phrase start)
+             iter#forward_chars (b2c buffer start)
```

Comment:
@herbelin This may be the cause of the problem Chris Dams mentioned in an email to Coq Club today.  I believe this code is used to populate the Errors panel.  If `start` is already relative to the beginning of the buffer, which seems likely, then this line should be:

```iter#forward_chars (byte_off_to_buffer_off phrase start)```

Can you tell me how to get errors to appear in the Errors panel so I can check this myself?

Reply:
Actually, the fix is in #15532.